google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/RestoredTableResult.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.admin.v2.models;
+
+import com.google.api.core.InternalApi;
+import com.google.common.base.Strings;
+import javax.annotation.Nullable;
+
+public class RestoredTableResult {
+
+  private final Table table;
+  private final OptimizeRestoredTableOperationToken optimizeRestoredTableOperationToken;
+
+  @InternalApi
+  public RestoredTableResult(
+      Table restoredTable, @Nullable String optimizeRestoredTableOperationName) {
+    this.table = restoredTable;
+    this.optimizeRestoredTableOperationToken =
+        Strings.isNullOrEmpty(optimizeRestoredTableOperationName)
+            ? null
+            : OptimizeRestoredTableOperationToken.of(optimizeRestoredTableOperationName);
+  }
+
+  public Table getTable() {
+    return table;
+  }
+
+  @Nullable
+  public OptimizeRestoredTableOperationToken getOptimizeRestoredTableOperationToken() {
```

Comment:
javadoc and in particular note why the token would be null here

Reply:
Done.